src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -791,6 +791,7 @@ RippleBlankingInputInTip = Ripple blanking input
 # ttl/TtlLibrary.java
 #
 ShowInternalStructure = Show the internal structure
+ttlInOutTip = In-Out %s
```

Comment:
`demultiplexerInTip` and `demultiplexerOutTip` use the full words "Input" and "Output", respectively. Therefore, consider to use "Input/Output" instead of "In-Out".

Reply:
Done.